.github/workflows/dogfood.yml

```
@@ -0,0 +1,50 @@
+name: dogfood merge
+# This workflow is triggered on pushes to master and dogfood branches
+on:
+  push:
+    branches:
+      - master
+      - 'dogfood/*'
+
+env:
+  SLACK_CHANNEL: squad-cloud-notifs
+  SLACK_USERNAME: Dogfood build action
+
+jobs:
+  dogfood_merge:
+    runs-on: ubuntu-latest
+    name: Update dogfood branch
+    permissions:
+      id-token: write # required for SonarSource/vault-action-wrapper
+    steps:
+      - name: get secrets
+        id: secrets
+        uses: SonarSource/vault-action-wrapper@v3
+        with:
+          secrets: |
+            development/kv/data/slack webhook | SLACK_WEBHOOK;
+            development/github/token/{REPO_OWNER_NAME_DASH}-dogfood-merge token | dogfood_token;
+      - name: git octopus step
+        env:
+          GITHUB_TOKEN: ${{ fromJSON(steps.secrets.outputs.vault).dogfood_token }}
+        id: dogfood
+        uses: SonarSource/gh-action_dogfood_merge@956e6de335578d0d301884414772b1bdb7234351
+        with:
+          dogfood-branch: 'dogfood-on-peach'
+      # Use the output from the `dogfood` step
+      - name: Get the name of the dogfood branch and its HEAD SHA1
+        run: echo "The dogfood branch was `${{ steps.dogfood.outputs.dogfood-branch }}` and its HEAD SHA1 was `${{ steps.dogfood.outputs.sha1 }}`"
+      # Slack notifications
+      - name: Notify success on Slack
+        uses: Ilshidur/action-slack@2.1.0
+        env:
+          SLACK_WEBHOOK: ${{ fromJSON(steps.secrets.outputs.vault).SLACK_WEBHOOK }}
+        with:
+          args: 'Dogfood build for `${{ steps.dogfood.outputs.sha1 }}`: *successful*'
+      - name: Notify failures on Slack
+        uses: Ilshidur/action-slack@2.1.0
+        if: failure()
+        env:
+          SLACK_WEBHOOK: ${{ fromJSON(steps.secrets.outputs.vault).SLACK_WEBHOOK }}
+        with:
+          args: 'Dogfood build for `${{ steps.dogfood.outputs.sha1 }}`: *failed*, see the logs at https://github.com/SonarSource/sonar-iac/actions'
```

```suggestion
          args: 'Dogfood build for `${{ steps.dogfood.outputs.sha1 }}`: *failed*, see the logs at https://github.com/SonarSource/sonar-go/actions'
```
Or, I think, you can also use `https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID`